src/utils/index.js

```diff
@@ -147,7 +147,8 @@ export function clearLines(grid) {
   
   return grid.reduce((result, row) => {
     if (!row.every(el => el !== null)) {
-      result.push(row)
+      let newRow = row.map(row => row)
+      result.push(newRow)
```

Comment:
why not `result.push([...row])`, which is much more compact

Reply:
= =